format_sd_card.sh

```diff
@@ -136,9 +136,20 @@ echo "| WARNING! WARNING! WARNING! WARNING! WARNING! WARNING! WARNING! |"
 echo "=================================================================="
 read ans
 
+DEVICE="$1"
+if [ "$DEVICE" = "vision-ai-rzv2l" ] ; then
+    format_sd_card_single_rootfs "$2"
+else
+	echo "Unsupported device $DEVICE"
+	exit 1
+fi
+
 DEVICE="$1"
 if [ "$DEVICE" = "smarc-rzg2l" ] ; then
 	format_sd_card_fat_and_rootfs "$2"
 elif [ "$DEVICE" = "smarc-rzv2l" ] ; then
 	format_sd_card_fat_and_rootfs "$2"
+else
```

Comment:
in both scripts, maybe move `elif [ "$DEVICE" = "vision-ai-rzv2l" ] ; then` here;

the way the script is setup now, it will work for `vision-ai-rzv2l` and won't work for `smarc-rzg2l` & `smarc-rzv2l`